### ISSUE TITLE
请问多列 in 查询怎么写

### ISSUE THREAD

**Issue by Merena** [Tencent/wcdb#1114]
swift中，请问select * from aTable where (a,b) in (values (1,2),(3,4)) 中，where的部分怎么改成WINQ呢

**Comment by Qiuwen-chen** [comment]
It's not a valid sql for SQLite.

**Comment by Merena** [comment]
网上说sqlite支持这么写，而且在SQliteStudio里能执行成功

https://blog.csdn.net/mzl87/article/details/125723853

![截屏2024-06-19 17 04 24](https://github.com/Tencent/wcdb/assets/10172720/1295ca4b-4a3c-46a0-8847-3270fc0dae4b)

![截屏2024-06-19 17 05 05](https://github.com/Tencent/wcdb/assets/10172720/a58fbd3c-d4ac-403e-87f7-9ccfafdb7b4c)

**Comment by Qiuwen-chen** [comment]
Indeed, it is not supported in Swift Winq yet. For now, you can convert it into multiple conditions.

**Comment by Merena** [comment]
OK，thank you.
